pkgconfig/libodp-dpdk.pc.in

```diff
@@ -3,9 +3,14 @@ exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
 
-Name: libodp-linux
+Name: libodp-dpdk
 Description: The ODP packet processing engine
 Version: @PKGCONFIG_VERSION@
+<<<<<<< HEAD:pkgconfig/libodp-linux.pc.in
 Libs: -L${libdir} -lodp-linux @DPDK_LIBS@
 Libs.private: @OPENSSL_STATIC_LIBS@ @DPDK_PMDS@ @DPDK_LIBS@ @PCAP_LIBS@ @PTHREAD_LIBS@ @TIMER_LIBS@ -lpthread @ATOMIC_LIBS@ @LIBCONFIG_LIBS@
+=======
+Libs: -L${libdir} -lodp-dpdk
+Libs.private: @ATOMIC_LIBS@
+>>>>>>> odpdpdk/master:pkgconfig/libodp-dpdk.pc.in
```

Comment:
This merge conflict is not addressed in [https://github.com/Linaro/odp/pull/123/commits/78eacc5eec31e41f82eba318d0030e8c3778bfed](url)